tabled/src/lib.rs

```diff
@@ -439,8 +439,8 @@ pub use crate::{tabled::Tabled, tables::Table};
 /// #[derive(Tabled)]
 /// struct Record {
 ///     #[tabled(skip)]
-///     id: [u8; 4],
-///     #[tabled(format("{}.{}.{}.{}.{}", self.id[0], self.id[1], self.id[2], self.id[3], self.name)]
+///     id: u8,
+///     #[tabled(format("{}.{}", self.id, self.name))]
```

Comment:
We do not support formatting using complex expressions like `self.id[0]` so I simplified this code a bit

Reply:
That's actually one of my endless (not released patches) :smile: 
I did rewrite this support more then a few times.

I wonder what do you think in this regard.
Shall we support functions calls etc?

```rust
#[tabled(format("{}.{}", some_function(self.id), self.name.one_more_field))]
id: u8,
```

Why I did rewrote it is cause it's impossible to support in inline enums :disappointed: 
And having 2 different approaches in 2 different places seeeems like not the right approach.

```rust
#[derive(Tabled)]
enum Something {
#[tabled(inline)]
A {
   #[tabled(format("{}", self.field1, func(0, 1, self)))]  // <- impossible to complex expressions with self 
   field1: usize,
},
B {}
}
```

So anyway maybe you have some thoughts

Reply:
What do you mean by rewriting? What do we have now if this is not possible to implement for inline structs? Is this the same functionality I assumed for the test fix? If so, I'm sure that supporting more advanced formatting functionality is a good feature, but obviously, there are many complications that can affect the decision of whether it's worth implementing or not.

If you have patches that extend this functionality beyond the scope of the changed test, what should this test look like to test as much of `#[tabled(format()]` as possible?